languages/php/README.md

```diff
@@ -1,100 +1,122 @@
 # Bitwarden Secrets Manager SDK wrapper for PHP
 
-PHP bindings for interacting with the [Bitwarden Secrets Manager]. This is a beta release and might be missing some functionality.
-Supported are CRUD operations on project and secret entities.
+PHP bindings for interacting with the [Bitwarden Secrets Manager]. This is a beta release and might be missing some
+functionality.
 
 ## Installation
 
-Requirements:
-- PHP >= 8.0
-- Composer
-- Bitwarden C libraries which you can generate using BitwardenSDK and following instructions in its readme (requires Rust). https://github.com/bitwarden/sdk
-If you are not using the standalone version of this library, file will be placed in `target/debug` folder if you are using from BitwardenSDK repository.
-- Access token for the Bitwarden account
-
+See the [installation instructions](./INSTALL.md)
 
 ## Usage
 
-To interact with the client first you need to obtain the access token from Bitwarden.
-You can then initialize BitwardenSettings passing $api_url and $identity_url if needed. These parameteres are
-optional and if they are not defined, BitwardenSettings instance will try to get these values from ENV, and
-if they are not defined there as well, it will use defaults: `https://api.bitwarden.com` as api_url and
-`https://identity.bitwarden.com` as identity_url. You can also pass device type as argument but that is entirely
-optional.
+### Create access token
 
-Passing BitwardenSettings instance to BitwardenClient will initialize it. Before using the client you must
-be authorized by calling the access_token_login method passing your Bitwarden access token to it.
+To interact with the client first you need to obtain the access token from Bitwarden.
+Review the help documentation on [Access Tokens].
 
+### Create new Bitwarden client
 
 ```php
-$access_token = '<your token here>';
-$api_url = "<api url>";
-$identity_url = "<identity url>";
+require_once 'vendor/autoload.php';
+
+$access_token = '<access-token>';
+$state_file = "<state-file>";
+$organization_id = "<organization-id>";
+$api_url = "https://api.bitwarden.com";
+$identity_url = "https://identity.bitwarden.com";
+
 $bitwarden_settings = new \Bitwarden\Sdk\BitwardenSettings($api_url, $identity_url);
 
 $bitwarden_client = new \Bitwarden\Sdk\BitwardenClient($bitwarden_settings);
-$bitwarden_client->access_token_login($access_token);
+$bitwarden_client->login_access_token($access_token, $state_file);
```

Comment:
```suggestion
$bitwarden_client->auth->login_access_token($access_token, $state_file);
```
(Should be `auth()`, but i made a suggestion in `BitwardenClient` to make it a property `auth`)

Reply:
Updated in https://github.com/bitwarden/sdk/pull/1032/commits/7bc8b9c08c672af458fd29f918177a28a39e2b01.